Cargo.toml

```diff
@@ -67,9 +71,10 @@ default = [
   "serde_bare",
   "serde_cbor",
   "serde_json",
-  "simd-json",
+  "simdjson",
   "speedy",
 ]
+simdjson = ["simd-json", "simd-json-derive"]
```

Comment:
I think we can make a `simd-json` feature that enables `dep:simd-json` and `simd-json-derive` instead.

Reply:
👍 I'll give tha ta shot :)